zigbee-api/src/main/java/org/bubblecloud/zigbee/network/discovery/AnnounceListenerImpl.java

```diff
@@ -79,6 +79,7 @@ public void run() {
                                 (short) zigbeeNetworkManager.getCurrentPanId()));
             }
         });
+        notifyThread.start();
```

Comment:
Since notifyThread is not used, I'd suggest removing the local variable, and simply adding .start() directly?